test/tests/src/constants.cpp

```diff
@@ -0,0 +1,338 @@
+#include <cassert>
+#include <chrono>
+#include "test.hpp"
+
+namespace CANlib {
+namespace map1 {
+extern Message_Type_T message_types[6];
+}
+namespace map2 {
+extern Message_Type_T message_types[28];
+}
+}  // namespace CANlib
+
+using namespace CANlib;
+using namespace std::chrono_literals;
+using map1::A_T;
+using map1::B_T;
+using map1::C_T;
+using map1::D_T;
+using map1::E_T;
+using map2::F_T;
+using map2::G_T;
+using map2::H_T;
+using map2::I_T;
+using map2::K_T;
+using map2::L_T;
+using map2::M_T;
+using map2::N_T;
+using map2::J::AA_T;
+using map2::J::BB_T;
+using map2::J::DD_T;
+using map2::J::EE_T;
+using map2::J::FF_T;
+using map2::J::GG_T;
+using map2::J::HH_T;
+using map2::J::II_T;
+using map2::J::JJ_T;
+using map2::J::KK_T;
+using map2::J::LL_T;
+using map2::J::MM_T;
+using map2::J::NN_T;
+using map2::J::CC::AAA_T;
+using map2::J::CC::BBB_T;
+using map2::J::CC::CCC_T;
+using map2::J::CC::DDD_T;
+
+void testKeys() {
+  map1::A_T A_input;
+  map1::B_T B_input;
+  map1::C_T C_input;
+  map1::D_T D_input;
+  map1::E_T E_input;
+  map2::F_T F_input;
+  map2::G_T G_input;
+  map2::H_T H_input;
+  map2::I_T I_input;
+  map2::J::AA_T AA_input;
+  map2::J::BB_T BB_input;
+  map2::J::CC::AAA_T AAA_input;
+  map2::J::CC::BBB_T BBB_input;
+  map2::J::CC::CCC_T CCC_input;
+  map2::J::CC::DDD_T DDD_input;
+  map2::J::DD_T DD_input;
+  map2::J::EE_T EE_input;
+  map2::J::FF_T FF_input;
+  map2::J::GG_T GG_input;
+  map2::J::HH_T HH_input;
+  map2::J::II_T II_input;
+  map2::J::JJ_T JJ_input;
+  map2::J::KK_T KK_input;
+  map2::J::LL_T LL_input;
+  map2::J::MM_T MM_input;
+  map2::J::NN_T NN_input;
+  map2::K_T K_input;
+  map2::L_T L_input;
+  map2::M_T M_input;
+  map2::N_T N_input;
```

Comment:
why are these all called `_input`?

Reply:
no particular reason, can be named in any way